### DISCUSSION TITLE
Priority Sync Question

### DISCUSSION THREAD

**Discussion by benkwan20** [airbytehq/airbyte#31513] [Connector Questions]
Hello Airbyte community,

I was wondering if there was some kind of way to make "priority syncs." For example, if I wanted to sync data from active campaigns in Google Ads, could I alter the connections to make that happen?

I tried to poke around, but was wondering if someone could answer that or send me in the right direction?

Best,

Ben

**Comment by marcosmarxm** [comment]
@benkwan20 can you elaborate a little more your use case?
For me priority sync would be to have different queues.
Per example: I have 100 connections but my setup can only trigger 10 sync in parallel and it's scheduled to have 15 syncs in a particular time window. The priority sync will be trigger first and after the others. But reading your text it doesn't look like this.

**Reply by benkwan20** [reply]
Thanks @marcosmarxm for your response! I will do my best. I think there are a couple of layers:
1. If I have 100 connections, I would like to sync 4 first (Google Ads, Klaviyo, Facebook Ads, and Shopify). The rest can be synced later.
2. Then, within those 4 first apps (Google Ads, Klaviyo, Facebook Ads, and Shopify), I would like to prioritize syncing specific current data.

Is this possible within Airbyte? I would like the most current data to be accessible within a couple of minutes and am wondering if there was a way to architect this with Airbyte as a solution.

[a comment by an automated account is not shown]
